streams/src/main/java/org/apache/kafka/streams/kstream/internals/graph/StreamsGraphNode.java

```diff
@@ -114,7 +139,7 @@ public String toString() {
         final String[] parentNames = parentNodeNames();
         return "StreamsGraphNode{" +
                "nodeName='" + nodeName + '\'' +
-               ", id=" + id +
```

Comment:
Shouldn't id be included ?

Reply:
Id is replaced with buildPriority.

Reply:
What about including valueChangingOperation and mergeNode here ?

Reply:
good point

Reply:
Should I send a minor PR ?

Reply:
I was going to do so, but if you want to submit a minor PR, that would be very helpful.  Thanks!

Reply:
See #5522